app/scripts/pages/files.js

```diff
@@ -33,36 +31,53 @@ default class Files extends React.Component {
     clearInterval(this.pollInterval)
   }
 
-  getFiles = () => {
+  getFiles () {
```

Comment:
No, the important part is that these are two different sorts of methods, if you declare them like this:

```js
getFiles () {
}
```

it's a regular method. If you declare it like this:

```js
```